ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendSpec.scala

```diff
@@ -12,12 +12,12 @@ import com.daml.ledger.resources.{Resource, ResourceContext}
 import com.daml.logging.{ContextualizedLogger, LoggingContext}
 import com.daml.metrics.Metrics
 import com.daml.platform.configuration.ServerRole
-import com.daml.platform.store.appendonlydao.DbDispatcher
```

Comment:
Try to use the `scalafmt` provided the repo in your IntelliJ formatter. It might help reduce the noise in the changes.